main.go

```diff
@@ -49,8 +52,31 @@ func internalServerError(err error, c echo.Context) error {
 	return c.JSON(http.StatusInternalServerError, ErrorResponse{Message: err.Error()})
 }
 
-func identityJSONTransform(j JsonLike) JsonLike {
-	return j
+func identityJSONTransform(body []byte) (interface{}, error) {
```

Comment:
not an identity transform anymore... give this a new name, like `jsonDeserialize`